common/flatpak-run.c

```diff
@@ -2941,6 +2942,24 @@ flatpak_run_setup_base_argv (FlatpakBwrap   *bwrap,
   else if (g_file_test ("/var/lib/dbus/machine-id", G_FILE_TEST_EXISTS))
     flatpak_bwrap_add_args (bwrap, "--ro-bind", "/var/lib/dbus/machine-id", "/etc/machine-id", NULL);
 
+  /* As per https://systemd.io/CONTAINER_INTERFACE parse os-release and pass through all
```

Comment:
At the time of writing, this is not part of the container interface, only a proposal.

Reply:
Just trying to be optimist :-D Changed to point to the RFE instead for now, if it all goes through I'll update it again back to the interface link.